src/core/issues-notes.js

```diff
@@ -118,7 +118,20 @@ function handleIssues($ins, ghIssues, conf) {
           }
         }
         $tit.find("span").text(text);
-        if (report.title) {
+        if (report.title && githubAPI) {
+          const labelsGroup = Array.from(ghIssue.labels)
+            .map(createLabels)
+            .reduce((frag, labelElem) => { 
```

Comment:
When in doubt: 

![screenshot 2018-03-19 13 33 25](https://user-images.githubusercontent.com/870154/37575236-207cf1fa-2b7a-11e8-8b85-e12edefa8ec5.png)

Where the sandwich is a `DocumentFragment` ;)

Reply:
Oh